FullCircle/FullCircle/Controllers/ActionListViewController/ActionListViewController.swift

```diff
@@ -80,7 +80,7 @@ class ActionListViewController: UIViewController {
     }()
     
     //MARK: - Internal Properties
-    var actions = [Action]() {
+    private var actions = [Action]() {
```

Comment:
Should remove private protection level here so we can use `actions` in the ActionListViewController extensions where the tableView delegate and datasource functions are @davidlawrencer

Reply:
fixed @msystang